src/main/java/Properties.java

```diff
@@ -176,6 +179,19 @@ public boolean hasLoadedProperties() {
         return !properties.isEmpty();
     }
 
+    public static boolean showErrorDialogue() {
+    	String prop = getInstance().properties.get("show_error_dialogue");
```

Comment:
Please do not use any abbriviations. There is no benefit in using short variables names. 
`prop` -> `properties`